integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -0,0 +1,103 @@
+"""
+Transmits consenting enterprise learner data to the Enterprise Integrated Channel SAP SuccessFactors.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import BaseCommand, CommandError
+
+from enterprise.models import EnterpriseCustomer
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+try:
```

Comment:
Any reason to do it this way, as compared to putting `django-celery` in the requirements and setting `CELERY_ALWAYS_EAGER` in the settings?